src/features/birthdayNotifications/components/EighteenBirthdayCard.tsx

```diff
@@ -1,30 +1,50 @@
 import React, { useState } from 'react'
+import styled from 'styled-components/native'
 
+import { useAuthContext } from 'features/auth/context/AuthContext'
 import { useBeneficiaryValidationNavigation } from 'features/auth/helpers/useBeneficiaryValidationNavigation'
-import { useDepositAmountsByAge } from 'shared/user/useDepositAmountsByAge'
 import TutorialPassLogo from 'ui/animations/eighteen_birthday.json'
 import { AchievementCardKeyProps, GenericAchievementCard } from 'ui/components/achievements'
+import { Spacer } from 'ui/components/spacer/Spacer'
+import { Typo } from 'ui/theme'
 
 export function EighteenBirthdayCard(props: AchievementCardKeyProps) {
   const [error, setError] = useState<Error | undefined>()
-  const depositAmount = useDepositAmountsByAge().eighteenYearsOldDeposit
-  const deposit = depositAmount.replace(' ', '')
+  const { user } = useAuthContext()
 
   const { navigateToNextBeneficiaryValidationStep } = useBeneficiaryValidationNavigation(setError)
 
   if (error) {
     throw error
   }
 
+  let text = 'Confirme tes informations personnelles pour débloquer tes 300\u00a0€.'
+  let buttonText = 'Confirmer mes informations'
+
+  if (user?.requiresIdCheck === true) {
+    text = 'Vérifie ton identité pour débloquer tes 300\u00a0€.'
+    buttonText = 'Vérifier mon identité'
+  }
+
+  const centerChild = () => (
```

Comment:
Habituellement, on met une majuscule à un composant React
```suggestion
  const CenterChild = () => (
```

Reply:
Après, l'idéal, c'est de donner un nom qui correspond plutôt au métier auquel ça correspond : à quoi sert ce texte dans la page ?